src/renderer/components/Visualiser/TopBar/GraqlEditor/GraqlCodeMirror.js

```diff
@@ -7,7 +7,7 @@ CodeMirror.defineSimpleMode('graql', {
   start: [
     { regex: /#.*/, token: 'comment' },
     { regex: /".*?"/, token: 'string' },
-    { regex: /(match|insert|delete|select|isa|sub|plays|relates|datatype|abstract|has|value|id|of|limit|offset|order|by|compute|from|to|in|aggregate|label|get|using|where)(?![-a-zA-Z_0-9])/, // eslint-disable-line max-len
+    { regex: /(match|insert|delete|select|isa|sub|plays|relates|value|abstract|has|value|id|of|limit|offset|order|by|compute|from|to|in|aggregate|label|get|using|where)(?![-a-zA-Z_0-9])/, // eslint-disable-line max-len
```

Comment:
we don't have aggregate anymore
`value` is in this list twice
 `select` isn't a keyword anymore (?)
`label` isn't a keyword anymore (we use `type`)

Reply:
addressed in https://github.com/graknlabs/workbase/pull/291/commits/fa0a2c1886642232b97fbaaf24ddec210c43a3e1

keywords are reduced to only those that are used in queries supported by Workbase (currently, `match get` and computer path`)